app/src/renderer/vuex/modules/wallet.js

```diff
@@ -39,7 +38,7 @@ export default ({ commit, node }) => {
       state.sendQueue.push(sendReq)
     },
     shiftSendQueue (state) {
-      state.sendQueue = state.sendQueue.shift(1)
+      state.sendQueue = state.sendQueue.length > 1 ? state.sendQueue.shift(1) : []
```

Comment:
Hm, looks like I had a bug when I originally implemented this. This line should be `state.sendQueue = state.sendQueue.slice(1)`, to shift off the first element (if any). (Also triggers a render since we're setting `sendQueue` to a different object.